Fix download test data when using ``pytest --pyargs rsciio -n``.